k8s/aat/common/ethos/ecm-consumer.yaml

```diff
@@ -0,0 +1,29 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: ecm-consumer
+  namespace: ethos
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:aat-*
+spec:
+  releaseName: ecm-consumer
+  rollback:
+    enable: true
+    retry: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: ecm-consumer
+    version: 0.0.1
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/ethos/ecm-consumer:aat-eb96e9bc
+      ingressHost: ecm-consumer-aat.service.core-compute-aat.internal
```

Comment:
this should be in the chart really and templated with the environment

Reply:
yeah it is in the chart, https://github.com/hmcts/ecm-consumer/blob/26fb1a5b18f557f81c4602fd2bd94dfcc8dc0402/charts/ecm-consumer/values.yaml#L4, i guess i can take that out from the aat and prod yaml right?

Reply:
yes from all of them

Reply:
updated!